Commands.md

```diff
@@ -737,6 +737,26 @@ If you wish to omit the config section, you may use `--no-config`:
 $ git info --no-config
 ```
 
+By default, the output is colored to make it easier on the eyes. When this is not desired (if piping the output of `git info` to another command, for example), the `--no-color` option may be used:
+
+```bash
+$ git info --no-color
+```
+
+You can select the number of previous commits to show by using the `-c <N>` option (where `<N>` is any integer). It is important to note, however, that `-c` **must be the FIRST option** you pass to `git info`. For example:
```

Comment:
We can use a while loop to get rid of this limitation, like: https://github.com/tj/git-extras/blob/master/bin/git-release#L31